source/egg/math/Matrix.cc

```diff
@@ -192,6 +192,16 @@ void Matrix34f::setBase(size_t col, const Vector3f &base) {
     mtx[2][col] = base.z;
 }
 
+/// @addr{0x806B41E0}
+void Matrix34f::setRotTangentHorizontal(const Vector3f &up, const Vector3f &tangent) {
+    EGG::Vector3f vec = tangent - up * tangent.dot(up);
+    vec.normalise2();
+
+    setBase(0, up.cross(vec));
+    setBase(1, up);
+    setBase(2, vec);
+}
+
```

Comment:
I'm no longer sure this should be defined in `Matrix34f`. Let's put this in `ObjectBase` alongside the other matrix function.